app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -0,0 +1,120 @@
+/* ********* Imports: ********* */
+package com.example.sabonit;
+import android.content.Context;
+import android.telephony.TelephonyManager;
+
+
+/**
+ * This class represents a user's account, which holds all the data of the user. The operations of
+ * the users with the database are based on the data stored in the account. Therefore the attribute
+ * phoneNumber needs to be unique, as a primary key. If a phone number stored in the google account
+ * and no phone number given as an input to the constructor, the phone number in the google account
+ * will be taken.
+ */
+public class Account
+{
+    /* ********* Constants: ********* */
+    // For validity phone number checking
+    private static int IMPLICIT_ISRAELI_PHONE_NUMBER_LENGTH = 9;
+    private static int ISRAELI_AREA_CODE = 13;
+    private static String ISRAELI_AREA_CODE_INFIX = "+972";
+
+
+    /* ********* Attributes: ********* */
+    // Given as an address
```

Comment:
Is the location street? city? give an example pattern in the code

Reply:
Probably a String of city and street, we didn't decide the little details (There is an option of creating an object of it)